### DISCUSSION TITLE
Issue to setup the application

### DISCUSSION THREAD

**Discussion by cjephuneh** [microsoft/call-center-ai#325] [Q&A]
HeIIo @clemlesne ,
Good evening from my side, My name is CaIeb Jephuneh and i want to set up CaII Center AI in my organisation. I have gone through the Readme on github but i dont seem to have success with running it.
If you can spare some time to take me through it over googIe meet i wiII be greatfuI and i would love to write a bIog about the process for other developers too.

I wiII be waiting to hear from you soonest.

**Comment by clemlesne** [comment]
Hello, what issue are you facing?

Can you provide:

- What are you trying to achieve
- Logs
- Steps you followed
- Application version

**Reply by cjephuneh** [reply]
sure i am running it on codespaces

**Comment by cjephuneh** [comment]
@clemlesne 
(.venv) @cjephuneh ➜ /workspaces/call-center-ai/app (main) $ python main.py
Traceback (most recent call last):
  File "/workspaces/call-center-ai/app/main.py", line 33, in <module>
    from app.helpers.call_events import (
ModuleNotFoundError: No module named 'app'

**Reply by clemlesne** [reply]
`python main.py` is not a documented startup command.

Please read the documentation: https://github.com/microsoft/call-center-ai?tab=readme-ov-file#local-on-your-machine

**Reply by cjephuneh** [reply]
noted!

**Comment by cjephuneh** [comment]
@clemlesne can you record a setup video? in engIish

**Reply by clemlesne** [reply]
The installation steps has been simplified with the `v12.0.0`.

Can you try to install the application to Azure (first) and confirm you succeed to do it? Don’t forget to use the latest version available.

**Comment by clemlesne** [comment]
Closing as no response for long. Don’t hesitate to reopen if needed.
